ndk/src/media/media_codec.rs

```diff
@@ -354,6 +545,28 @@ impl MediaCodec {
         MediaError::from_status(status)
     }
 
+    pub fn input_buffer(&self, index: usize) -> Option<&mut [u8]> {
```

Comment:
Can `InputBuffer::buffer_mut()` now simply call `self.codec.input_buffer(self.index)`? Would like to deduplicate things as much as possible.

Same for `OutputBuffer::buffer()` calling `self.codec.output_buffer(self.index)`.